### PR TITLE
Update site directory names

### LINKED ISSUE #122: Amend language site directory names, redirects
The decision's been made to go with two-letter country codes for our language site sub-directories.

- [x] @Pete-Robelou, can we please rename our `en-GB` (and `jp-JP`, for the purposes of demonstration?) directories to their two-letter equivalent? `en` and `jp`, respectively. Expect we'll also need to amend any reference of `en-GB` where used for default/fallback locale, or collection paths, etc.
- [x] @djgalloway Once this has been done, can we please get your support in updating the redirects to reflect the change? If that's just a case of changing redirect references in `/ansible/roles/deploy/templates/site.j2` (?) then @Pete-Robelou, we can manage that as part of our code updates.

In future, should we need region-specific language identifiers for language sites (e.g. `pt-br`), we'll introduce them as necessary, and for those sites only.

### PR DESCRIPTION
**Pull request by Pete-Robelou**
- en-GB becomes en
- ja-JP becomes jp

Closes #122

### CODE REVIEW

**Review by adamduncan — APPROVED** [review]
Nice one, @Pete-Robelou. Looks nice and tidy with the two-letter alpha codes 👍

**Comment by djgalloway** [comment]
jenkins test this please

**Comment by djgalloway** [comment]
jenkins test now please?

**Comment by djgalloway** [comment]
https://language-site-directory-names.ceph.io/en/

File: src/en/news/blog/search-output.html [hunk 1]
```diff
@@ -4,6 +4,6 @@
 override:tags: []
 ---
 
-[ {% for item in collections['ja-JP-blog-post'] %} { "image":"{{item.data.image}}", "title":"{{item.data.title}}",
```
Comment (djgalloway) [inline src/en/news/blog/search-output.html:7]: Was this supposed to get changed to `en`?

File: src/jp/news/blog/search-output.html [hunk 2]
```diff
@@ -4,6 +4,6 @@
 override:tags: []
 ---
 
-[ {% for item in collections['en-GB-blog-post'] %} { "image":"{{item.data.image}}", "title":"{{item.data.title}}",
```
Comment (djgalloway) [inline src/jp/news/blog/search-output.html:7]: Ah.. or maybe github got confused about which files to rename

Comment (adamduncan) [reply src/jp/news/blog/search-output.html:7]: Yeah, good spot @djgalloway. That caught my eye too. Ran the build locally and all was looking good. I think GitHub got muddled with the rename

Comment (djgalloway) [reply src/jp/news/blog/search-output.html:7]: Out of pure curiosity, @Pete-Robelou, did you use `mv` or `git mv` locally?

Comment (Pete-Robelou) [reply src/jp/news/blog/search-output.html:7]: I didn't use either. I manually updated all file/folder names and any syntax within them.

Everything is as it should be on my local machine but like you say, I guess GitHub mixed up the `news/blog/search-output.html` files.

Am I correct in saying that it still works as the `en-GB/.../search-output.html` was renamed to `jp/.../search-output.html` and `ja-JP/.../search-output.html` was renamed to `en/.../search-output.html` ?

Comment (djgalloway) [reply src/jp/news/blog/search-output.html:7]: Yeah I'm sure it still works fine.  Just odd.

**Comment by adamduncan** [comment]
Hey @djgalloway — just looking at the develop site after this [latest build](https://jenkins.ceph.com/view/all/job/ceph-website/246/) went out. All looking successful on deployment. 

Though when I navigate to https://develop.ceph.io I'm being redirected to https://develop.ceph.io/en-GB/ where there's now a 404. Just wanted to check if there wasn't something we needed to prod in the direction of the new default directory: https://develop.ceph.io/en/

Thank you

**Comment by djgalloway** [comment]
Ah, yes.  Updating the nginx config is not automatic.  I'll do that now.

**Pull request merged**